packages/flutter/lib/src/material/data_table.dart

```diff
@@ -47,6 +47,10 @@ class DataColumn {
   /// [Icon] (typically using size 18), or a [Row] with an icon and
   /// some text.
   ///
+  /// By default, this widget will only occupy the minimal space. If you want
+  /// it to take the entire remaining space, e.g. when you want to use [Center],
```

Comment:
Would using an Expanded be the only way to center the label? Would it be possible to set a size of space to take up?

Reply:
Sorry if I misunderstood your question. The problem here is the widget structure, which is roughly:
```
TableCell
 |- Row
      |- label
      |- arrow & padding
```
the default behavior is to align everything left, including the label and the arrow (right if numerical), and the only thing the developer can control is `label`.

We can't provide an `Expanded` by default since it changes the behavior to aligning label left and arrow right.

Reply:
Gotcha!